flink-runtime/src/main/scala/org/apache/flink/runtime/jobmanager/JobManager.scala

```diff
@@ -173,7 +173,7 @@ class JobManager(
    * to run in the actor system of the associated job manager.
    */
   val webMonitorPort : Int = flinkConfiguration.getInteger(
-    ConfigConstants.JOB_MANAGER_WEB_PORT_KEY, -1)
+    JobManagerOptions.WEB_PORT.key(), -1)
```

Comment:
I thought about doing that, but decided against it to preserve existing behavior. If no port is configured at all, with the current code, a random port will be used. Someone out there may be relying on this (although they should just explicitly set it to 0)